src/app/components/Card/Card.tsx

```diff
@@ -40,13 +40,8 @@ export default function Card({
       </div>
       <section>
```

Comment:
Please add a small padding to this padding to create some space between the border and the text/rating